roles/openshift-emptydir-quota/tasks/main.yaml

```diff
@@ -3,7 +3,7 @@
   replace:
     dest: /etc/origin/node/node-config.yaml
     regexp: '^(.*)perFSGroup: (\s+.*)?$'
-    replace: '\1 perFSGroup: 512Mi\2'
+    replace: '\1 perFSGroup: {{ openshift_node_local_quota_per_fsgroup }}\2'
```

Comment:
It looks like it is in the installer https://github.com/openshift/openshift-ansible/blob/master/roles/openshift_node/templates/node.yaml.v1.j2#L58 so yeah, I agree on deprecate it on every provider